cases/dfi/namelist.input.3

```diff
@@ -0,0 +1,127 @@
+ restart=.true.,
+ restart_interval=9
+ run_days                            = 0,
+ run_hours                           = 0,
+ run_minutes                         = 0,
+ run_seconds                         = 0,
+ start_year                          = 2016, 2016, 2000,
+ start_month                         = 03,   03,   01,
+ start_day                           = 23,   23,   24,
+ start_hour                          = 00,   00,   00,
+ start_minute                        = 09,   09,   00,
+ start_second                        = 00,   00,   00,
+ end_year                            = 2016, 2016, 2000,
+ end_month                           = 03,   03,   03,
+ end_day                             = 23,   23,   25,
+ end_hour                            = 00,   00,   00,
+ end_minute                          = 12,   12,   00,
+ end_second                          = 00,   00,   00,
+ interval_seconds                    = 10800
+ input_from_file                     = .true.,.true.,.true.,
+ frames_per_outfile                  = 1, 1, 1000,
+ history_interval                    = 3,    3,    60,
+ io_form_history                     = 2
+ io_form_restart                     = 2
+ io_form_input                       = 2
+ io_form_boundary                    = 2
+ debug_level                         = 0
+ write_hist_at_0h_rst                = .true.,
+ /
+
+ &domains
+ auto_levels_opt=2
+ dzstretch_s=1.3
+ dzstretch_u=1.1
+ e_vert=45,45,45
+ time_step                           = 180,
+ time_step_fract_num                 = 0,
+ time_step_fract_den                 = 1,
+ max_dom                             = 2,
+ e_we                                = 75,    88,   94,
+ e_sn                                = 70,    79,    91,
+ p_top_requested                     = 5000,
+ num_metgrid_levels                  = 27,
+ num_metgrid_soil_levels             = 4,
+ dx                                  = 30000, 10000,  3333.33,
+ dy                                  = 30000, 10000,  3333.33,
+ grid_id                             = 1,     2,     3,
+ parent_id                           = 0,     1,     2,
+ i_parent_start                      = 1,     25,    30,
+ j_parent_start                      = 1,     22,    30,
+ parent_grid_ratio                   = 1,     3,     3,
+ parent_time_step_ratio              = 1,     3,     3,
+ feedback                            = 1,
+ smooth_option                       = 0
+ /
+
+ &physics
+ num_land_cat=21
+ physics_suite = 'CONUS'
+ radt                                = 30,    30,    30,
+ bldt                                = 0,     0,     0,
+ cudt                                = 5,     5,     5,
+ isfflx                              = 1,
+ ifsnow                              = 1,
+ icloud                              = 1,
+ surface_input_source                = 1,
+ num_soil_layers                     = 4,
+ sf_urban_physics                    = 0,     0,     0,
+ /
+ 
+&dfi_control
+ dfi_opt                             = 3,
```

Comment:
On the restart nml, we turn the dfi off, right? This was just a cursory check, I may have missed some subtleties.